examples/getting_started/2-Customization.ipynb

```diff
@@ -42,6 +42,7 @@
    "source": [
     "import pandas as pd\n",
     "import holoviews as hv\n",
+    "from holoviews import opts\n",
```

Comment:
Commenting for the code three cells down, I think we should switch from ``group`` to ``label`` there if we're not using the group for anything:

```python
curve = hv.Curve(spike_train, 'milliseconds', 'Hertz', label='Firing Rate')
spikes = hv.Spikes(spike_train.sample(300), 'milliseconds', [], label='Spike Train')
curve + spikes
```

Reply:
Fixed in 550d9e61